spec/features/editing_images/choose_lead_image_spec.rb

```diff
@@ -24,14 +26,15 @@ def given_there_is_an_edition_with_images
     @edition = create(:edition,
                       document_type_id: document_type.id,
                       image_revisions: [@image_revision])
+
+    @publishing_api_request = stub_publishing_api_put_content(@edition.content_id, {})
```

Comment:
I wonder if we want this to be just a stub_any_publishing_api_put_content since we assert the specific request in the "preview_creation_succeeded" step so looks like we're double checking this.